src/ng-select/ng-select.component.ts

```diff
@@ -268,6 +268,7 @@ export class NgSelectComponent implements OnInit, OnDestroy, OnChanges, AfterVie
 
     setDisabledState(isDisabled: boolean): void {
         this.isDisabled = isDisabled;
+        this.detectChanges();
```

Comment:
add unit test

Reply:
Ok, I do this quickly

Reply:
Done, hope it's ok for you ;)

Reply:
@Skrytch thanks for the fix, I simplified test in #295